types/react-resize-detector/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-resize-detector 2.2
+// Type definitions for react-resize-detector 3.1
 // Project: https://github.com/maslianok/react-resize-detector
 // Definitions by: Matthew James <https://github.com/matthew-matvei>, James Greenleaf <https://github.com/aMoniker>
```

Comment:
@rdrgn You should perhaps add yourself to the list of authors

Reply:
Added. Thank you for reviewing.